src/app/features/public-form/edit-publicform-route/edit-publicform-route.component.scss

```
@@ -1,29 +1,60 @@
 @use "variables/breakpoints";
 @use "variables/colors";
 
+.form-link-container {
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
+}
+
 .readonly-prefix {
   word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)